doc/rbd/libvirt.rst

```diff
@@ -111,11 +111,12 @@ To configure Ceph for use with ``libvirt``, perform the following steps:
 	admin socket = /var/run/ceph/$cluster-$type.$id.$pid.$cctid.asok
 
    The ``client.libvirt`` section name should match the cephx user you created
-   above.
+   above.  
    If SELinux or AppArmor is enabled, note that this could prevent the client
    process (qemu via libvirt) from doing some operations, such as writing logs
    or operate the images or admin socket to the destination locations (``/var/
-   log/ceph`` or ``/var/run/ceph``).
+   log/ceph`` or ``/var/run/ceph``). Additionally, make sure that the libvirt
+   and libvirt-qemu users have appropriate access to the specified directory.
```

Comment:
There is no such thing as `libvirt-qemu` (on Fedora at least)

Reply:
@dillaman 
On ubuntu os, the right user is libvirt-qemu
```
liuc@aep-test02:~$ sudo grep 'libvirt\|qemu' /etc/group
libvirt:x:122:bmpa,,liuc
libvirt-qemu:x:64055:libvirt-qemu
```
I'll refine the patch and  add an example for Ubuntu in the doc

Reply:
I don't want to call out specific users for special OSs. In reality, people should understand file permissions. If you really want to keep this, just keep it generic as "Additionally make sure that the directory has appropriate permissions".

Reply:
@dillaman 
Follow your initial suggestion. Thanks for your review.